docs/API_SPECS_DEVICE_MODELS.md

```diff
@@ -485,4 +485,4 @@ DELETE /v1/devicemodels/${id}
 Response:
 ```
 200 OK
-```
\ No newline at end of file
+```
```

Comment:
what is the change here? if nothing has changed then revert to master version for this file to keep history clean

Reply:
done